src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20190715_add_internal_resource_id.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="mbemis" id="add-cluster-internal-resource-id">
+        <addColumn tableName="CLUSTER">
+            <column name="internalId" type="VARCHAR(254)">
```

Comment:
i would have liked to enforce that this is unique at the database-level but given that the IDs generated in the below sql command won't be unique, that's a no-go without a more thorough migration

Reply:
Yeah, I agree with this reasoning.